include/zephyr/bluetooth/l2cap.h

```diff
@@ -557,6 +558,52 @@ int bt_l2cap_ecred_chan_connect(struct bt_conn *conn,
  */
 int bt_l2cap_ecred_chan_reconfigure(struct bt_l2cap_chan **chans, uint16_t mtu);
 
+/** @brief Reconfigure Enhanced Credit Based L2CAP channels
+ *
+ *  Experimental API to reconfigure L2CAP ECRED channels with explicit MPS and
+ *  MTU values.
```

Comment:
Do we have a way to track experiment function APIs like this, in a way that can help us remove the experimental note later?

Reply:
I think so far we have promoted APIs from experimental on request. I think that makes sense. The [API lifecycle process requires multiple applications to use an API before it's promoted, to "unstable"](https://docs.zephyrproject.org/latest/develop/api/api_lifecycle.html#hardware-agnostic-apis).

Reply:
Yeah, but do have any way to track individual functions? In generally the API lifecycle usually covers entire groups for functions like `bt_*`. 

We have defined versions to provide information about stable/unstable/experiemental as defined by https://docs.zephyrproject.org/latest/develop/api/overview.html, but we rarely use that for individual functions and mostly for groups of functions.